test/unit/ping.test.ts

```diff
@@ -44,232 +41,201 @@ describe('Ping', () => {
     jest.clearAllTimers();
   });
 
-  it('accepts null options', () => {
-    const ping = new Ping({} as any);
-    expect(ping).toBeTruthy();
-  });
-
-  it('when started it sends a ping on an interval', () => {
-    let ping = new Ping(client, standardOptions);
-
-    ping.start();
-    jest.advanceTimersByTime(PING_INTERVAL_WITH_BUFFER);
-    expect(pingCallCount).toBe(1);
-    jest.advanceTimersByTime(PING_INTERVAL_WITH_BUFFER);
-    expect(pingCallCount).toBe(2);
-  });
-
-  it('when started multiple times it sends a ping on a single interval', () => {
-    let ping = new Ping(client, standardOptions);
-
-    ping.start();
-    ping.start();
-    ping.start();
-    jest.advanceTimersByTime(PING_INTERVAL_WITH_BUFFER);
-    expect(pingCallCount).toBe(1);
-    ping.start();
-    jest.advanceTimersByTime(PING_INTERVAL_WITH_BUFFER);
-    expect(pingCallCount).toBe(2);
+  describe('constructor()', () => {
+    it('accepts null options', () => {
+      const ping = new Ping({} as any);
+      expect(ping).toBeTruthy();
+    });
```

Comment:
I didn't actually have to change any of the tests. Just restructured them and removed an old one that no longer applies with my new changes. I guess that is one good thing about writing IT tests, they should still pass when you make some changes :D